engine/helpers_test.go

```diff
@@ -472,8 +472,8 @@ func TestGetCollatedExchangeAccountInfoByCoin(t *testing.T) {
 
 	info.Exchange = "Bitstamp"
 	info.Accounts = append(info.Accounts,
-		exchange.Account{
-			Currencies: []exchange.AccountCurrencyInfo{
+		account.SubAccount{
+			Currencies: []account.Balance{
```

Comment:
with what i was telling u earlier, theres nothing wrong with this test, its just that we are appending ltc's sub account on top of btc's so the 1st info consists of: {Bitfinex [{ [{BTC 100 0}]}]} &  2nd info consists of: {Bitstamp [{ [{BTC 100 0}]} { [{LTC 100 0}]}]}.
can u just change it so its more visible/obvious to someone who might be looking at it the first time